Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-black = "*"
+black = "==24.1.1"
```

Comment:
We don't need to lock package here, since it is already locked in Pipfile.lock. Or was it specific reason of doing it?

Reply:
I think we do need that :) Pipfile should contain the version hints if we care about the version of particular dependency. In case of black we do care and do not want to update it basically ever. I would argue that for some other we also care about not jumping to the next major versions but I fixed just black since it's the most obvious. I changed it in this PR because I was actually adding docker sdk. I later removed it because it sucked and used other way to run docker (See here: https://github.com/NordSecurity/rust_build_utils/pull/12) but this change is still beneficial so I left it in.